pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.feedzai</groupId>
     <artifactId>pdb</artifactId>
-    <version>2.8.7-SNAPSHOT</version>
+    <version>0.0.0-SNAPSHOT</version>
```

Comment:
I'm guessing this jgitver will automatically discover the next SNAPSHOT version when we do a materialize git commit, right?

Reply:
Not the git commit. It's based on the tags.